docs/source/notebooks/tutorial_residuals.ipynb

```diff
@@ -0,0 +1,1067 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "4c01ab6b-ea9b-4bff-8c6f-abac185b6936",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "hide-input"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import seaborn as sns\n",
+    "\n",
+    "sns.set_theme(\n",
+    "    rc={\"axes.spines.top\": False, \"axes.spines.right\": False, \"figure.autolayout\": True},\n",
+    "    style=\"ticks\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85b86cd8-0b55-4cb4-a7b2-28a7d55eb9e5",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Plotting simulations & residuals spectra"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d17b0825-d1ae-46b5-a8b3-6941f0355a9b",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "Simulations and data as well as covariance are stored as `sacc` objects. For more details regarding `sacc` format, you can have a look to the tutorial [notebooks](https://github.com/simonsobs/sacc/tree/master/examples) or you can refer to the official [documentation](https://sacc.readthedocs.io/en/latest/). Here we will read the power spectra for all the simulations and compare its mean value to theory + foregrounds (without and with systematics). \n",
```

Comment:
The link to [the sacc notebook tutorial](https://github.com/simonsobs/sacc/tree/master/examples) does not work for me